### ISSUE TITLE
Merge 1.4.0 ?

### ISSUE THREAD

**Issue by akofman** [nolanlawson/relational-pouch#48]
Hello,

I just noticed that from the npm repo, the last version of this project is 1.4.0 which is different from the master branch. Would it be possible to merge it to master ?

I can't find any branch for this version, only a tag and it's not really convenient in order to compare, fork or fix properly some issues.

Thanks !

**Comment by nolanlawson** [comment]
I'm sorry, but it looks like I accidentally published to npm without actually pushing to Github. Big error on my part; I'll have to reconstruct what I did and hopefully push out a 1.4.1. Super sorry about that. :/

**Comment by akofman** [comment]
No worries ! Super thanks for your reply ;)

**Comment by nolanlawson** [comment]
1.4.1 is out now.

**Comment by akofman** [comment]
Just to be sure, is it normal to still have some differences comparing with tag v1.4.0 ? 
https://github.com/nolanlawson/relational-pouch/compare/master...v1.4.0

**Comment by nolanlawson** [comment]
1.4.1 was just based off of the master branch, plus rebuilt with latest dependencies. I literally have no idea what went into 1.4.0, because it's lost now.

**Comment by akofman** [comment]
Ok perfect. I just asked because I noticed a `db.rel.query` method in 1.4.0 that does not exist anymore in 1.4.1.

**Comment by nolanlawson** [comment]
That was almost certainly a mistake, because I don't think I intentionally merged the change to add `db.rel.query` (it's not even documented). Hopefully nobody tried to use it!

**Comment by akofman** [comment]
Haha I almost did :p
So in order to query and keep relations, what would be your guidance ?

**Comment by nolanlawson** [comment]
Dunno. I don't even understand where `db.rel.query` came from. Someone added support for `pouchdb-find`, but I'm hesitant because I think relational-pouch isn't really optimized to solve that same set of problems.

**Comment by akofman** [comment]
You mean that would produce kind of performance issues ?

**Comment by nolanlawson** [comment]
I'm mostly concerned about tightly coupling two separate plugins.

Also any implementation of `db.rel.query` or `db.rel.find` could easily exist as a separate plugin, so I kinda feel like that's where it should go.

**Comment by akofman** [comment]
Hmmm ok I can understand your hesitation. In my case, I'm using both plugins and it's fine to keep them separate. Adding the glue to pouchDB-find is simple anyway.

Just a last question if you don't mind :p 
Assuming I have a schema with many-to-many relations between authors and books (let's say a book could have been written by multiple authors, and an author could have written multiple books).
I noticed that if I request a particular author with the find method : `db.rel.find('authors', 1)` ,  If one of its books has two related authors, the result will be an array of two authors. 

So I'm requesting only one author but at the end I get an array of two authors and I have to iterate between them to retrieve the one I really want. I hope I'm clear enough ...

So I just wanted to know if it is the normal behaviour or if I did miss something ?
